packages/serverpod_shared/test/password_manager_test.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:serverpod_shared/serverpod_shared.dart';
+import 'package:test/test.dart';
+import 'package:yaml/yaml.dart';
+
+const _defaultPasswordConfig = '''
+shared:
+  mySharedPassword: 'my password'
+
+development:
+  database: 'development db pass'
+  redis: 'development redis pass'
+
+  serviceSecret: 'development service secret'
+
+staging:
+  database: 'staging db pass'
+  serviceSecret: 'staging service secret'
+
+production:
+  database: 'production db pass'
+  serviceSecret: 'production service secret'
+''';
+
+void main() {
+  group(
+      'Given default password config when loading Map in runMode development then',
+      () {
+    const runMode = 'development';
+
+    var passwords = PasswordManager(runMode: runMode)
+        .loadPasswordsFromMap(loadYaml(_defaultPasswordConfig));
+
+    test('Passwords are not null', () {
```

Comment:
Nit: lower case letter on the first word when you put them in a group:

The log output will be:
`Given default password config when loading Map in runMode development then Passwords are not null`